interface/main/messages/save.php

```diff
@@ -0,0 +1,256 @@
+<?php
+/**
+ * /interface/main/messages/save.php
+ *
+ * Copyright (C) 2017 MedEx <support@MedExBank.com>
+ *
+ * LICENSE: This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * @package OpenEMR
+ * @author MedEx <support@MedExBank.com>
+ * @link http://www.open-emr.org
+ */
+
+
+require_once("../../globals.php");
+require_once("$srcdir/acl.inc");
+require_once("$srcdir/lists.inc");
+require_once("$srcdir/api.inc");
+require_once("$srcdir/formatting.inc.php");
+require_once("$srcdir/forms.inc");
+require_once("$srcdir/patient.inc");
+require_once("$srcdir/MedEx/API.php");
+//require_once(dirname(__FILE__). "/log.inc");
+
+$MedEx = new MedExApi\MedEx('MedExBank.com');
+//you need admin privileges to update this.
+if ($_REQUEST['go'] =='Preferences') {
+	$query = "SELECT * FROM users where id = ?";
+	$user_data = sqlQuery($query,array($_SESSION['authUserID']));
+
+ 	if (acl_check('admin', 'super')) {
+		$sql = "UPDATE `medex_prefs` set `ME_facilities`=?,`ME_providers`=?,`ME_hipaa_default_override`=?,
+			`PHONE_country_code`=? ,`MSGS_default_yes`=?,
+			`POSTCARDS_local`=?,`POSTCARDS_remote`=?,
+			`LABELS_local`=?,`LABELS_choice`=?,
+			`combine_time`=?";
+
+		$facilities 	= implode("|",$_REQUEST['facilities']);
+		$providers 		= implode("|",$_REQUEST['providers']);
+		$HIPAA 			= ($_REQUEST['ME_hipaa_default_override'] ? $_REQUEST['ME_hipaa_default_override'] : '');
+		$MSGS 			= ($_REQUEST['MSGS_default_yes'] ? $_REQUEST['MSGS_default_yes'] : '');
+		$country_code 	= ($_REQUEST['PHONE_country_code'] ? $_REQUEST['PHONE_country_code'] : '1');
+
+		$myValues = array($facilities,$providers,$HIPAA,$country_code,$MSGS,$_REQUEST['POSTCARDS_local'],$_REQUEST['POSTCARDS_remote'],$_REQUEST['LABELS_local'],$_REQUEST['chart_label_type'],$_REQUEST['combine_time']);
+
+	    $_GLOBALS['chart_label_type'] = $_REQUEST['chart_label_type'];
+	    sqlStatement( 'UPDATE `globals` set gl_value = ? where gl_name like "chart_label_type" ', array( $_REQUEST['chart_label_type'] ) );
```

Comment:
If you are using one type of label, why should you have to change it more than once?  If others are printing labels in other places across the system, I am presuming they are printing the same type of label there also.  Seems logical that if you need a new label type, why not set it here and save the steps you'd have to do to change it way over there in globals?  If it is already set in the global correctly, then you do not need to change it here.  It made sense to me to save the steps.